processor/src/lib.rs

```diff
@@ -99,6 +99,21 @@ pub mod crypto {
 
 type QuadFelt = QuadExtension<Felt>;
 
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub struct MemoryAddress(u32);
+
+impl MemoryAddress {
+    pub fn as_u32(&self) -> u32 {
+        self.0
+    }
```

Comment:
nit: I would change this to be a `From<MemoryAddress> for u32` implementation instead

Reply:
Done.